Core/CategoriesManager.cs

```diff
@@ -11,25 +11,26 @@ public static class CategoriesManager
     {
 		public static Dictionary<string, Func<Transaction, bool>> Load()
 		{
-			if (!File.Exists("categories.json"))
+			//TODO: load from user provided file
+			if (true)
```

Comment:
Codacy found an issue: [Remove this useless 'if' statement.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)